packages/icons/scripts/icons.js

```diff
@@ -20,16 +17,41 @@ fetchImages({
 }).then(async svgs => {
   console.log("Building icons...");
 
-  const jsx = await Promise.all(svgs.map(svg => svgtojsx(svg.buffer)));
-  const data = svgs
-    .map((svg, index) => `export const ${pascalCase(svg.name)}Icon = () => ${jsx[index]}\n`)
+  const allIconExports = (
+    await Promise.all(
+      svgs.map(svg =>
+        transform(
+          svg.buffer.toString(),
+          {
+            icon: 20,
+            replaceAttrValues: {
+              black: "currentColor",
+              "#383838": "currentColor",
+              "#38383D": "currentColor",
```

Comment:
Ooo, this is interesting. Too bad svgr doesn't support pattern matching here for hex codes. Maybe we can add a verification step into our script that looks over the generated TypeScript and warns if there are any hard-coded colors that we missed?

Reply:
I'm moving from svgr to svgo and some simple string parsing so it will just use what's declared in Figma now, this way we can do a regex for these 😎. I also cleaned them up in the Figma file since they were a little messy.

Reply:
Didn't end up needing regex here since svgo takes care of removing fill so we can control it at the top-level.